android/autodispose-androidx-lifecycle/src/androidTest/java/autodispose2/androidx/lifecycle/AndroidLifecycleScopeProviderTest.java

```diff
@@ -43,11 +43,11 @@ public void observable_beforeCreate() {
     final PublishSubject<Integer> subject = PublishSubject.create();
 
     // Spin it up
-    TestLifecycleOwner lifecycle = TestLifecycleOwner.create();
+    TestLifecycleOwner lifecycle = new TestLifecycleOwner();
```

Comment:
The default `initialState` is `Lifecycle.State.STARTED` - you'll want to use `new TestLifecycleOwner(LifecycleState.INITIALIZING)` if you want to manually move through each lifecycle state.

Same for all other uses.

Reply:
neat!